editor/src/components/navigator/navigator.spec.browser2.tsx

```diff
@@ -5049,7 +5049,7 @@ describe('Navigator', () => {
         'regular-sb/scene/pg:dbc',
         'regular-sb/scene/pg:dbc/78c',
         'render-prop-sb/scene/pg:dbc/78c/prop-label-header-header',
-        'regular-sb/scene/pg:dbc/78c/pro', // <- the inserted render prop
```

Comment:
seems like by adding the to-be-inserted uid to `uniqueIds.add(uid)` we make it so that the inserted uid can _never_ appear in the final project because the fixUtopiaElement will treat the UID as taken.

is this unavoidable?